finders/bixi/parser.py

```diff
@@ -1,24 +1,40 @@
 from typing import List
 import json
 from finders.gps import GPSPosition
-from finders.stops import Stop, BixiStop
+from finders.stops import BixiStopInformation, BixiStopStatus
+import requests
+from requests.exceptions import HTTPError
 
 
-def parse_bixi_station_information(json_str: str) -> List[BixiStop]:
-    # list of stops to return
-    stops_list = []
+def get_json_from_http(url):
```

Comment:
This function returns None when get is unsuccessful but nothing is documented 
Either raise an exception and do not catch it or return a boolean for sucess